adminlogin.aspx

```diff
@@ -47,7 +47,7 @@
                                  </div>
 
                                   <div class="mb-3">
-                                      <asp:Button class="btn btn-primary btn-success btn-block btn-lg" ID="Button1" runat="server" Text="Login" />
+                                      <asp:Button class="btn btn-primary btn-success btn-block btn-lg" ID="Button1" runat="server" Text="Login" OnClick="Button1_Click" />
```

Comment:
### Bad naming convention

What is this button supposed to do? Think about yourself and the team. We want to be able to look at code and read it and know what it does with minimal effort. 

```suggestion
                                      <asp:Button class="btn btn-primary btn-success btn-block btn-lg" ID="btnExample" runat="server" Text="Login" OnClick="Button1_Click" />
```